src/scripts/config/main.ts

```diff
@@ -78,5 +78,6 @@ export default {
       'http://twitter.com/intent/tweet?text=ESA%20Climate%20From%20Space&url={currentUrl}'
   },
   legendImage: `${baseUrlStorage}/legend-images/{variable}.png`,
-  downloadUrls
+  downloadUrls,
+  localStorageLanguage: 'language'
```

Comment:
It's the key and not the language itself. Please name it 'localStorageLanguageKey'